src/Source/WMSSource.js

```diff
@@ -0,0 +1,76 @@
+import Source from './Source';
+import URLBuilder from '../Provider/URLBuilder';
+
+const supportedFormats = [
```

Comment:
Is this still relevant ?

Reply:
I moved this part in `Source`. But, I think it's not respond to the question.
could you precise?

Reply:
removed